vyper/semantics/types/function.py

```diff
@@ -469,16 +469,13 @@ def fetch_call_return(self, node: vy_ast.Call) -> Optional[BaseTypeDefinition]:
                 if not isinstance(kwarg.value, vy_ast.NameConstant):
                     raise InvalidType("skip_contract_check must be literal bool", kwarg.value)
             else:
-                if not isinstance(kwarg.arg, vy_ast.VyperNode):
-                    raise ArgumentException(
-                        (
-                            "Usage of kwarg in Vyper is restricted to gas=, "
-                            "value= and skip_contract_check="
-                        ),
-                        kwarg.value,
-                    )
-                # To preserve for future kwargs (?)
-                validate_expected_type(kwarg.arg, kwarg.value)
+                raise ArgumentException(
+                    (
+                        "Usage of kwarg in Vyper is restricted to gas=, "
+                        "value= and skip_contract_check="
+                    ),
+                    kwarg.value,
```

Comment:
```suggestion
                    kwarg.arg,
```

Reply:
There is a slight issue with using `kwarg.arg` here. Since `kwarg.arg` is of `str` type instead of `VyperNode` type, the code snippet is not printed out when the exception is raised. 

`kwarg.value` on the other hand is of `VyperNode` type.

Reply:
ah, i think then `kwarg`